stingray/events.py

```diff
@@ -228,10 +228,11 @@ def from_lc(lc):
 
         return EventList(time=times, gti=lc.gti)
 
-    def simulate_times(self, lc, use_spline=False, bin_time=None):
-        """
-        Randomly assign (simulate) photon arrival times to an :class:`EventList` from a
-        :class:`stingray.Lightcurve` object, using the acceptance-rejection method.
+    def simulate_times(self, lc, use_spline=False):
```

Comment:
This removes a keyword argument in a user-exposed function, no? Does that mean this will break backwards compatibility?

Reply:
Right. I'll add it back with a deprecation warning or something